Detector/DetFCChhECalSimple/compact/FCChh_ECalBarrel_Mockup.xml

```diff
@@ -4,40 +4,23 @@
   xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
 
   <info name="FCChh_ECalBarrel_Mockup"
-    title="ECal Barrel Calorimeter"
-    author="A.Dell'Acqua"
+    title="Simple ECal Barrel Calorimeter"
+    author="A.Dell'Acqua, J.Faltova"
     url="no"
     status="development"
     version="1.0">
-    <comment>ECal very conceptual design</comment>
+    <comment>
+      Most simple liquid argon - lead Ecal design.
+      It consists of cylindrical layers of lead and liquid argon.
+      It includes cryostat.
+    </comment>
   </info>
 
   <define>
-    <constant name="BarECalOuterSpacer" value="100*mm"/>
-    <constant name="BarECalCryoThickness" value="100*mm"/>
-    <!-- calorimeter volume -->
-    <constant name="BarCryoECal_rmin" value="BarECal_rmin+BarECalOuterSpacer"/>
-    <constant name="BarCryoECal_rmax" value="BarECal_rmax-BarECalOuterSpacer"/>
-    <constant name="BarCryoECal_dz" value="BarECal_dz"/>
-
-    <constant name="EMBarrel_rmin" value="BarCryoECal_rmin+BarECalCryoThickness"/>
-    <constant name="EMBarrel_rmax" value="BarCryoECal_rmax-BarECalCryoThickness"/>
-    <constant name="EMBarrel_dz" value="BarECal_dz-BarECalCryoThickness"/>
+    <include ref="FCChh_ECalBarrel_CrystatThickness.xml" />
+    <constant name="CryoThickness" value=CommonCryoThickness/>
```

Comment:
Obviously it seems to work like this, but can we change to `"CommonCryoThickness"` for consistency?